src/material-examples/menu-icons/menu-icons-example.html

```diff
@@ -1,4 +1,4 @@
-<button mat-icon-button [matMenuTriggerFor]="menu">
+<button mat-icon-button [matMenuTriggerFor]="menu" aria-label="Example icon">
```

Comment:
This could be 
```
Example icon-button with a menu
```

Reply:
Hi jelbourn, "menu" is getting appended  by default so the complete label becomes "example icon menu". I tested that. Still do you want me to change it to "example icon- button with a" so that it will become "example icon-button with a menu" ?

Reply:
I'm guessing what you're hearing is the screen-reader describing the element. If that's the case, it would still be appropriate to include the whole `Example icon-button with a menu`  would still be appropriate because it's explaining the content separately from what the control _is_.